### DISCUSSION TITLE
GitHub Sponsor - Join Waitlist button disappeared

### DISCUSSION THREAD

**Discussion by D3vil0p3r** [community/community#23065] [New to GitHub]
<p>Hello,<br>
I have a personal profile and an organization profile on my account for managing a specific project. Today I explored GitHub Sponsor and I joined the Waitlist for my organization profile and for my personal profile. At the end, I clicked on “Delete GitHub Sponsor account” for my Personal Profile but now, when I go to <a href="https://github.com/sponsors/accounts" class="inline-onebox" rel="noopener nofollow ugc">Sign in to GitHub · GitHub</a> I dont see my personal profile with the Join the Waitlist button anymore, it disappeared.<br>
I was not aware that the Delete button could also prevent the join to the Waitlist…</p>
<p>If in the future I would like to set a Sponsorship account for my personal profile, how can I do it?</p>

**Comment by laurie** [comment, marked as answer]
<p>Hi there <a class="mention" href="/d3vil0per">@D3vil0per</a> :wave:</p>
<p>You’re welcome to get in touch with support as and when you want to join the waitlist again in future, and we can revert your Sponsors profile to that stage.</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://support.github.com/request/sponsors">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/29237130/181074649-db93f259-75de-4908-8e30-e66c71e30017.png" class="site-icon" width="32" height="32">

      <a href="https://support.github.com/request/sponsors" target="_blank" rel="noopener">support.github.com</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://support.github.com/request/sponsors" target="_blank" rel="noopener">Sponsors - GitHub Support</a></h3>

  <p>GitHub Support is here to help. Learn about GitHub products, browse our helpful resources, and contact support with your questions.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Hope that helps! :smiley_cat:</p>
